recipes/default.rb

```diff
@@ -46,15 +47,13 @@
   group 'root'
   mode '700'
   recursive true
-  action :create
 end
 
 directory '/var/lib/ganeti/rapi' do
-  owner 'root'
-  group 'root'
+  owner 'gnt-rapi'
+  group 'gnt-masterd'
```

Comment:
So, as I said, this only need to happen on Ganeti >= 3.0.0. We should make a helper method that uses the info from the following to get the version of the installed version:
```
[root@gprod1.osuosl.bak ~]# gnt-cluster version
Software version: 2.16.1
Internode protocol: 2160000
Configuration format: 2160000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.16.1
```

Reply:
Should this use the `node['ganeti']['version']` attribute to pull the version instead of shelling out?

Reply:
@detjensrobert hmm, that would work but we'd need to update osl-ganeti to set that (as we use a local attribute currently). We can assume `nil` is `>= 3.0`